### ISSUE TITLE
Linux Tage Chemnitz (Germany) [16./17. March] (speaker name: Isabel)

### ISSUE THREAD

**Issue by MaineC** [InnerSourceCommons/InnerSourceMarketing#294]
# Immediately
- [x ] Fill in the conference name and dates in the title.
- [ ] Assign this issue to yourself.

# 4 Weeks Prior
- [ ] Introduce yourself in the [#marketing] channel and ask for some ISC stickers to be shipped to you.
- [ ] Create a new channel in the [InnerSource Commons Slack] specifically for your conference.
Format is `#event-[conference_name]-[year]`.
- [ ] Let people know in the [#general] channel about the conference.
Encourage them to attend and to join your new Slack channel
- [ ] Plan some type of in-person get-together for ISC attendees.
Prefer this to happen as early on as you can.
It could be as simple as meeting at a specified location for a meal or chat.
- [ ] Let everyone know in the event channel what is the get-together plan.

# 2 Weeks Prior
- [ ] Remind everyone in your Slack channel of the get-together plan.
- [ ] Remind people in the [#general] channel about the conference and your Slack channel.

# 1 Week Prior
- [ ] Remind everyone in your Slack channel of the get-together plan.
- [ ] Pack the ISC stickers.
- [ ] Remind people in the [#general] channel about the conference and your Slack channel.

# Conference Day(s)
- [ ] Remind everyone in your Slack channel of the get-together plan.
- [ ] Hold the get-together.
- [ ] Hand out ISC stickers wherever you can.
- [ ] If you meet anyone feels InnerSource-related, invite them to join the [InnerSource Commons Slack].
- [ ] Have fun at the conference!

# Week After
- [ ] Share back in the [#marketing] channel how your experience went.
Better, yet, attend a [#marketing] working group meeting and share on the call!
- [ ] Archive your Slack channel.

[#general]: https://app.slack.com/client/T04PXKRM0/C04PXKRN4
[InnerSource Commons Slack]: https://app.slack.com/client/T04PXKRM0
[#marketing]: https://app.slack.com/client/T04PXKRM0/CUWFGQJ8K

**Comment by MaineC** [comment]
Created issue as requested by @rrrutledge , the purpose is to make transparent that my talk in Chemnitz was accepted.

I left the rest of the issue template as is, though it seems a bit too involved for just a talk at a conference.

**Comment by rrrutledge** [comment]
@MaineC do you have some stickers to give out?

**Comment by MaineC** [comment]
No, I don't.

**Comment by rrrutledge** [comment]
Thank you for going and representing InnerSource and the InnerSource Commons!

**Comment by MaineC** [comment]
Adding my summary from Slack here so it's acutally kept:

I wanted to give a brief summary of my experience in Chemnitz
For those not living in Germany: Chemnitz is a mid-sized town in south-east Germany/ Sachsen. The event takes place in the university. The crowd it draws tends to be a mixture of junior developers as well as older people, but without the usual Linux Foundation/ international Open Source background. There are professionals, coming from both, mid-sized companies to large businesses like T-Systems and SAP. It's a lot of locals, but also people coming with the Linux bus from Cologne. In addition there's several people visiting the event while visiting their parents living in the area. Except for two talks the entire schedule was in German.
The talk was scheduled for 12:00/ lunch time on day 2 (Sunday, there you have the enthusiasts sitting in a uni lecture hall on a sunny Sunday, some daddies with their children) right after a talk on how political players try to game the patch submission process of the Linux kernel for their state propaganda (speaker coming from SuSE).
The lecture hall was roughly half full. People seemed pretty interested in the topic. I scoped the talk to cover the benefits for the employer and ended with a note on which benefits InnerSource brings for becoming a better Open Source citizen.
The audience seemed to be able to follow the argument just fine. People in the audience were happy about the benefits that InnerSource brings already to their own processes. One question was interesting - related to how to balance following the "need to know" principle in GDPR/ data privacy and the "we want to open all the sources" that an ideal InnerSource world would love to see.
However: The most shocking learning for me going back to an audience that isn't in the usual circles that we interact with on a daily basis: Even gray beards who obviously have spent their entire life in IT were unaware of the term "foundation". Even talking to a group of people who go to an Open Source event like Linux Tag the people there had no clue that as a first step they could go to our web page, understand why the should get involved with our little community - let alone how to do that.



To summarize:
I do believe it makes sense to spread the word at smaller, more local events - in particular where that can be done at low or no cost.
When doing that though - even when it's an Open Source event - there's a good chance that people need a lot of hand holding to actually get active and participate here.
